lib/beaker/hypervisor/vagrant.rb

```diff
@@ -53,6 +53,7 @@ def make_vfile(hosts, options = {})
       hosts.each do |host|
         host.name.tr!('_', '-') # Rewrite Hostname with hyphens instead of underscores to get legal hostname
         set_host_default_ip(host)
+        v_file << "  c.ssh.keep_alive = true\n" if host['ssh_keep_alive'] == true
```

Comment:
oh that's neat. I'm wondering if we should enable it by default/always?

Reply:
Personaly i thik it would be a safe enough change to make this the default but will wait to see if others have a view